oozie/src/main/java/org/apache/falcon/oozie/DatabaseExportWorkflowBuilder.java

```diff
@@ -94,12 +96,15 @@ private String buildSqoopCommand(Cluster cluster, Feed feed) throws FalconExcept
         StringBuilder sqoopArgs = new StringBuilder();
         StringBuilder sqoopOptions = new StringBuilder();
 
-        buildConnectArg(sqoopArgs, cluster).append(ImportExportCommon.ARG_SEPARATOR);
-        buildTableArg(sqoopArgs, cluster).append(ImportExportCommon.ARG_SEPARATOR);
-        Datasource datasource = DatasourceHelper.getDatasource(FeedHelper.getExportDatasourceName(
-                FeedHelper.getCluster(entity, cluster.getName())));
-        ImportExportCommon.buildUserPasswordArg(sqoopArgs, sqoopOptions, datasource)
-                .append(ImportExportCommon.ARG_SEPARATOR);
+        org.apache.falcon.entity.v0.feed.Cluster feedCluster = FeedHelper.getCluster(entity, cluster.getName());
+        Datasource datasource = DatasourceHelper.getDatasource(FeedHelper.getExportDatasourceName(feedCluster));
```

Comment:
How do we handle connector specific direct mode and verbose options from the properties (which are key value pairs but these are single arguments to Sqoop)